### ISSUE TITLE
Docker hub versions make no sense

### ISSUE THREAD

**Issue by jonathanstiansen** [jwilder/nginx-proxy#579]
Hey JWilder, really appreciate the work you've done. I like to version my companies images they are using, so I tried to look into the history of your project [here](https://hub.docker.com/r/jwilder/nginx-proxy/tags/).

The dates and numbers make no sense you have a `latest` tag which doesn't correspond in size to any other version (it should be corresponding to something else). And you have a version `0.3.0` made 3 days ago and a version `0.3.6` made 10 months ago. 

Could you follow, maybe semantic versioning? Just increase the last digit every commit you push - it would be awesome :-D

(this came up because I couldn't get one of your new versions to work, I'll file a different but for that)

Again sorry to ask more of you. Thank you for your work.

**Comment by jwilder** [comment]
I do use semantic versioning.  The images are built using docker hub's automated builds and I have no idea when it rebuilds them. The tags on the hub correspond to tags on this repo.  `0.1.0` is built off of the `0.1.0` tag.  `latest` is just the `HEAD` on `master`.  `0.3.6` was a tag corresponding to the version of `docker-gen` from a long time ago and I didn't realize it was still around.  I've removed that.  No idea what `master` is or how it got there, but I also removed that.

If you use `latest`, you running what ever is committed to `master`.  `0.4.0` would be the latest stable version.  `0.5.0` will be next.
